src/components/AudioPlayer/AudioPlayer.stories.mdx

```diff
@@ -0,0 +1,166 @@
+import { action } from "@storybook/addon-actions";
+import {
+  ArgsTable,
+  Canvas,
+  Description,
+  Meta,
+  Story,
+} from "@storybook/addon-docs";
+import { withDesign } from "storybook-addon-designs";
+
+import AudioPlayer from "./AudioPlayer";
+import { getCategory } from "../../utils/componentCategories";
+import DSProvider from "../../theme/provider";
+
+<Meta
+  title={getCategory("AudioPlayer")}
+  component={AudioPlayer}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=36835%3A25338",
+    },
+    jest: ["AudioPlayer.test.tsx"],
+  }}
+  argTypes={{
+    audioType: {},
+    className: { control: false },
+    id: { control: false },
+  }}
+/>
+
+# Audio Player
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `1.2.0`    |
+| Latest            | `1.2.0`    |
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Component Props](#component-props)
+- [Accessibility](#accessibility)
+- [Example Embed Code Snippets](#example-embed-code-snippets)
+- [Errored](#errored)
+
+## Overview
+
+<Description of={AudioPlayer} />
```

Comment:
I would remove this.  Right now, the description coming in from the `.tsx` file is not offering anything more than what is described in the text below.